### ISSUE TITLE
we should show the source for dart: libraries when debugging with dartdevc

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/sdk#29891]
When debugging with DDC, we show the source for user and package code (via source maps). We should do the same for dart: libraries, for parity, and it is helpful to see the original source when debugging.

It looks like all the dart: source is compiled into dart_sdk.js. There is a sourcemap entry at the end:

```
//# sourceMappingURL=dart_sdk.js.map
```

I don't know if we ship with an actual sourcemap file, if the source Dart libraries would need to be processed before compilation in order to create a better source map, or if pub would need work to serve up the dart: references.

@vsmenon @jakemac53

**Comment by devoncarew** [comment]
(looks like there is a `dart_sdk.js.map`)

**Comment by jakemac53** [comment]
Duplicate of https://github.com/dart-lang/pub/issues/1623

**Comment by jakemac53** [comment]
tldr; the sourcemaps we get today don't work with pub serve. We have to either change the sourcemaps or change pub serve to understand them.
